recipes/arabic_reshaper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "arabic_reshaper" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 008d71e6f5523833d8f4c8b5fd487d8bbed444c7c9a06665695afdb5ef47f214
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
```

Comment:
Please double check that setup tools is needed as a runtime dependency, it is usually only needed at build time.

Reply:
I actually only found it it is needed when the tests failed on the CI system. I already proposed a PR to the original project to add `setuptools` to their requirements in `setup.py`. So to answer your request: yes, I'm absolutely sure it's a runtime requirement.